packages/contracts/contracts/factory/RepoFactory.sol

```diff
@@ -0,0 +1,92 @@
+/*
+ * SPDX-License-Identifier:    MIT
+ */
+
+pragma solidity 0.8.10;
+
+import "../utils/Proxy.sol";
+import "../registry/APMRegistry.sol";
+import "../APM/Repo.sol";
+
+/// @title RepoFactory to create a Repo
```

Comment:
Why do we need a Repo factory? Couldn't the Repo contract itself have helper methods to assist this?

Reply:
for separation of concern.
but most importantly because of contract code size limit: https://eips.ethereum.org/EIPS/eip-170